Dockerfile

```diff
@@ -8,4 +8,11 @@ RUN pip install -r requirements.txt
 ADD . /code/
 ADD wait.sh /usr/local/bin/
 RUN chmod +x /usr/local/bin/wait.sh
+
+RUN groupadd rescue-kerela && \
```

Comment:
The images are only used for local dev and not deployed at all. So it is unnecessary to add groups/users.